### PR TITLE
Add JDK14 to pipelines

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- Remove refs to jdk12
- Reference repos
- Docker slaves
- PBT-All
- Variable file

[skip ci]
Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
@jdekonin for review.
Marking as draft until jdk14 repo is created.

**Comment by pshipton** [comment]
> until jdk14 repo is created.

Scheduled for Dec 12th.

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -157,203 +159,209 @@ s390x_linux:
...
     11:
       ? special.system
-    12:
-      ? special.system
     13:
       ? special.system
+    14:
+      ? special.system
+    next:
+      ? special.system
```
Comment (jdekonin) [inline buildenv/jenkins/variables/defaults.yml]: We don't have the machine availability to handle these extra tests.  At least not until we have more AIX brought online.

Comment (jdekonin) [reply buildenv/jenkins/variables/defaults.yml]: Or maybe this is irrelevant as we don't run these test mightly/daily.

Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: We don't generally run `next` testing iirc. I was just adding the exclude to be more consistent.

**Comment by AdamBrousseau** [comment]
Depends #8098 

Jenkins compile plinux jdk8,jdk11

File: buildenv/jenkins/variables/defaults.yml [hunk 2]
```diff
@@ -538,46 +329,36 @@ x86-64_windows:
...
     11:
       - special.system
-    12:
-      - special.system
     13:
       - special.system
+    14:
+      - special.system
+    next:
+      - special.syste
```
Comment (pshipton) [inline buildenv/jenkins/variables/defaults.yml]: missing an 'm' on system

Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: fixed

**Comment by AdamBrousseau** [comment]
Jenkins compile plinux jdk8,jdk11

**Comment by AdamBrousseau** [comment]
Was using the 8/11 build to prove the change to `jdk_image_dir`
Doesn't work because `set_jdk_folder()` doesn't use buildspec manager. Will move this out to a different PR.

**Comment by AdamBrousseau** [comment]
Jenkins test sanity.functional plinux jdk14

**Comment by pshipton** [comment]
The compile failures (like `error: cannot access VarForm) are something that needs to be fixed for jdk14 independent of this change.

**Comment by pshipton** [comment]
Jenkins test sanity.functional plinux jdk14

**Comment by pshipton** [comment]
Something is wrong, there is a directory name missing
`/Build_JDK14_ppc64le_linux_Personal@tmp/durable-12659866/script.sh: build/linux-ppc64le-server-release/images//bin/java: not found`

**Comment by AdamBrousseau** [comment]
I think I missed it because I originally had the changes from #8135 in this PR.
Fixed now. Also removed the excludes for 13+.
Jenkins test sanity.functional plinux jdk14

File: buildenv/jenkins/variables/defaults.yml [hunk 3]
```diff
@@ -106,60 +106,47 @@ largeheap:
...
     next: '--disable-ddr'
   extra_make_options: 'EXTRA_CMAKE_ARGS="-DOMR_WARNINGS_AS_ERRORS=FALSE"'
   excluded_tests:
       - *defaultTests
       - special.system
 #========================================#
+# JITServer
+#========================================#
+jitserver:
+  extra_configure_options: '--enable-jitserver=yes'
```
Comment (pshipton) [inline buildenv/jenkins/variables/defaults.yml]: Pls remove the `=yes` as per #8196

File: buildenv/jenkins/variables/defaults.yml [hunk 4]
```diff
@@ -286,163 +200,71 @@ s390x_linux_xl:
...
-    13: '--openssl-version=1.1.1d'
-    next: '--openssl-version=1.1.1d'
+    build: 'ci.role.build && hw.arch.x86 && sw.os.cent.6'
   extra_configure_options:
-    8: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
-    11: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched --enable-jitserver=yes'
-    12: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched'
-    13: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched'
-    next: '--enable-cuda --with-cuda=/usr/local/cuda-9.0 --with-openssl=fetched'
+    all: '--enable-cuda --with-cuda=/usr/local/cuda-9.0'
```
Comment (pshipton) [inline buildenv/jenkins/variables/defaults.yml]: This should include `--enable-jitserver`, assuming #8191 is merged first.

File: buildenv/jenkins/variables/defaults.yml [hunk 5]
```diff
@@ -600,59 +356,39 @@ x86-32_windows:
   excluded_tests:
     8:
       - special.system
-    11:
```
Comment (pshipton) [inline buildenv/jenkins/variables/defaults.yml]: This shouldn't be adding Windows jdk11 special.system testing.

**Comment by pshipton** [comment]
Note the copyright check failures

File: buildenv/jenkins/variables/defaults.yml (hunk 5 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: This is win32 so it doesn't apply which is why I am removing the exclude.

File: buildenv/jenkins/variables/defaults.yml (hunk 3 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: Done via #8098

File: buildenv/jenkins/variables/defaults.yml (hunk 4 above)
Comment (AdamBrousseau) [reply buildenv/jenkins/variables/defaults.yml]: Based on https://github.com/eclipse/openj9/pull/8191#issuecomment-571298582 I will leave as-is for now. Please advise if you'd like it included or if 8191 is merged first.

**Comment by AdamBrousseau** [comment]
Updated copyrights and rebased again on master

**Comment by pshipton** [comment]
Jenkins compile plinux jdk14

**Pull request merged**